packages/web-components/src/components/cbp-list/cbp-list.tsx

```diff
@@ -10,9 +10,15 @@ export class CbpList {
   
   private renderedTag: HTMLElement;
 
+  /** Specifies the variant of list (simple, link, special) */
+  @Prop() variant: 'simple' | 'link'; 
```

Comment:
needs to be reflected if you're using it as a CSS hook. You're still missing the "unstyled" list as well. I would remove "simple" and just list the actual variants of "link" and "unstyled".